dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -136,15 +136,22 @@ const decideMediaType = (format: ArticleFormat): MediaType | undefined => {
 	}
 };
 
-const decideKicker = (trail: FEFrontCard) => {
+const decideKicker = (
+	trail: FEFrontCard,
+	cardInTagFront: boolean,
```

Comment:
This could have also been `cardInPressedFront` but it made sense to me that the default case of this method is whatever comes from `kickerText` which is in the pressed fronts model. Happy to rename though if this doesn't make sense!